### PR TITLE
0.7.3

### LINKED ISSUE #47: no primary key defined in lookml for page views

### LINKED ISSUE #56: Update package readme
* New package syntax for installation
* Link to new docs site in line 29
* etc.

### PR DESCRIPTION
**Pull request by jtcohen6**
### Changelog

* Fix whitespace issue caused by pass-through column code in #53
* Small updates to lookml: fixes #47 
* Update README: fixes #56

This PR ended up being a bit of a grab bag, so that we have a solid + stable next release, before some more foundational/structural changes in 0.8.0.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
-dbt data models for snowplow analytics. Adapted from Snowplow's [web model](https://github.com/snowplow/web-data-model)
+dbt data models for sessionizing Snowplow data. Adapted from Snowplow's [web model](https://github.com/snowplow/web-data-model).
 
-### models ###
+### Models ###
 
-The primary data models contained in this package are described below. While other models exist,
-they are primarily used to build the two primary models listed here.
+The primary ouputs of this package are **page views** and **sessions**. There are
+several intermediate models tasked with created these two.
```
Comment (drewbanin) [inline README.md]:
```suggestion
several intermediate models used to create these two models.
```

File: README.md [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
 ```
 
-- run `dbt deps`.
+- Run `dbt deps` to install the package.
 
-### configuration ###
+## Configuration ###
 
-The [variables](http://dbt.readthedocs.io/en/master/guide/context-variables/#arbitrary-configuration-variables) needed to configure this package are as follows:
+The [variables](https://docs.getdbt.com/docs/using-variables) needed to configure this package are as follows:
```
Comment (drewbanin) [inline README.md:30]: 🙏

File: README.md [hunk 3]
```diff
@@ -36,8 +37,10 @@ The [variables](http://dbt.readthedocs.io/en/master/guide/context-variables/#arb
 |snowplow:context:web_page|Schema and table for web page context|Yes|
 |snowplow:context:performance_timing|Schema and table for perf timing context, or `false` if none is present|Yes|
 |snowplow:context:useragent|Schema and table for useragent context, or `false` if none is available|Yes|
+|snowplow:pass_through_columns|Additional columns for inclusion in final models|Yes|
```
Comment (drewbanin) [inline README.md]: This should be optional, right? The default is `[]`

File: README.md [hunk 4]
```diff
@@ -52,94 +55,50 @@ models:
...
-    ) returns timestamptz
-as $$
-declare
-begin
-  return in_t at time zone out_tzname at time zone in_tzname;
-end;
-$$ language plpgsql;
-```
+These models were originally written for Redshift, with added support for
+Snowflake and BigQuery. To run on Postgres, create [these UDFs] in your database.
```
Comment (drewbanin) [inline README.md]: is this missing a link?

File: README.md [hunk 5]
```diff
@@ -52,94 +55,50 @@ models:
...
-...
+### Database support
 
-repositories:
-  - "git@github.com:fishtown-analytics/snowplow.git"
-```
+* Redshift
+* Snowflake
+* BigQuery
+* Postgres, with the creation of [these UDFs](pg_udfs.sql) in your database
```
Comment (drewbanin) [inline README.md]: does this link go to the right place?

File: macros/adapters/default/sessions/snowplow_sessions_tmp.sql [hunk 6]
```diff
@@ -66,12 +66,12 @@ prep AS (
         sum(time_engaged_in_s) as time_engaged_in_s,
 
         max(case when last_page_view_in_session = 1 then page_url else null end)
-            as exit_page_url,
+            as exit_page_url
         
         {%- for column in var('snowplow:pass_through_columns') %}
-        max(case when last_page_view_in_session = 1 then {{column}} else null end)
+        , max(case when last_page_view_in_session = 1 then {{column}} else null end)
```
Comment (drewbanin) [inline macros/adapters/default/sessions/snowplow_sessions_tmp.sql:72]: i didn't notice this in the original PR, but should this be `last_page_view_in_session` or `first_page_view_in_session`? I imagine first, right?

Comment (jtcohen6) [reply macros/adapters/default/sessions/snowplow_sessions_tmp.sql:72]: We include both `first_` and `last_` for the custom column, which is par with the [Segment package](https://github.com/fishtown-analytics/segment/blob/master/macros/cross-adapter-modeling/sessionization/segment_web_sessions__initial.sql#L49). The `prep` CTE here gives us access to `last_`, whereas we can just pull `first_` from the first page view (`a` alias) in the join below.

**Review by drewbanin — APPROVED** [review]
LGTM! Can you include a brief summary of the changes present in the Release Notes when you create a new release?

**Pull request merged**